### DISCUSSION TITLE
Replacing banana with another food

### DISCUSSION THREAD

**Discussion by AshTrayBY** [Vdauphin/HeartsAndMinds#1560] [Question and answer] [answered]
Hi,
want to change ACE_banana to another consumable food so players will be able to share their own food with civilians. Changed ACE_banana to ACE_Can (consumable food from another mod fully compatible with ace/acex food system) but civilians not taking with new food. Changes were made at add_actions.sqf, foodGive.sqf and foodRemove.sqf

**Comment by Vdauphin** [comment]
Hello,
Does `ACE_Can` is a class name?
Cheers

**Comment by AshTrayBY** [comment]
> Hello, Does `ACE_Can` is a class name? Cheers

Sure.
Also wondering is there a way to add a list of items to give to civilians as food?

**Reply by Vdauphin** [reply, marked as answer]
> Sure.

Are you sure, otherwise it should work

> Also wondering is there a way to add a list of items to give to civilians as food?

I didn't design it like that. The main idea was to give candy to civilian, not really feed them

**Reply by AshTrayBY** [reply]
ACE_Can is working as banana. Tested wrong mission version without changes
